src/optimizer/remove_unused_columns.cpp

```diff
@@ -187,6 +187,15 @@ void RemoveUnusedColumns::VisitOperator(LogicalOperator &op) {
 		remove.VisitOperator(*op.children[0]);
 		return;
 	}
+	case LogicalOperatorType::LOGICAL_UPDATE: {
```

Comment:
This is the same as the INSERT above, so we can simplify this with switch statement fall-through, e.g.:

```cpp
case  LogicalOperatorType::LOGICAL_INSERT:
case LogicalOperatorType::LOGICAL_UPDATE:
   ...
```